client/components/data/query-home-cards/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestHomeCards } from 'state/home/actions';
```

Comment:
Should we rename to `requestHomeLayout` (since I've renamed the route to `/home/layout` in D40630-code)?

Reply:
5780180ae1dbc11bd1dd7677c01b7230235902f5